tests/integration/test_workflow_requirements_integration.py

```
@@ -0,0 +1,221 @@
+        pr_agent_file = WORKFLOWS_DIR / "pr-agent.yml"
+        
+        if not pr_agent_file.exists():
+            pytest.skip("pr-agent.yml not found")
+        
+        with open(pr_agent_file, 'r') as f:
+            workflow = yaml.safe_load(f)
+        
+        # Check that at least one job has a step to install dependencies
+        has_install_step = False
+        
+        for job_name, job in workflow.get('jobs', {}).items():
```

Ruff (0.14.6)

Loop control variable `job_name` not used within loop body

Rename unused `job_name` to `_job_name`

(B007)